packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -246,7 +248,7 @@ class CreateCommand extends CreateBase {
       windows: featureFlags.isWindowsEnabled && platforms.contains('windows'),
       windowsUwp: featureFlags.isWindowsUwpEnabled && platforms.contains('winuwp'),
       // Enable null safety everywhere.
-      dartSdkVersionBounds: '">=2.12.0 <3.0.0"',
+      dartSdkVersionBounds: '">=$dartSdk <3.0.0"',
```

Comment:
Using the string from your example comment above, will pub be able to resolve:

`>=2.1.0 (build 2.1.0-dev.8.0 4312ae32 <3.0.0`

Reply:
No, this is meant to add just the `2.1.0-dev.8.0` part.

If you compare the two getters in `cache.dart`, you see that:
   * `dartSdkVersion` returns: `'${match[1]} (build ${match[1]}$noFlutter)';`
   * `dartSdkBuild` returns: `'${match[1]}$noFlutter';`
   
The template uses the latter.

 So I think this will give us just the raw Dart build number?

Reply:
Ahh yes, when I tried this locally it looks correct. Can you add unit tests for `dartSdkBuild` to https://github.com/christopherfujino/flutter/blob/549683d56dea35e10aa4ebd62df846f544406d65/packages/flutter_tools/test/general.shard/cache_test.dart, including using the stable dart release patterns?

Reply:
Add a test; was it something like that you had in mind?